exporter/awscloudwatchlogsexporter/README.md

```diff
@@ -27,6 +27,10 @@ exporters:
   awscloudwatchlogs:
     log_group_name: "testing-logs"
     log_stream_name: "testing-integrations-stream"
+    sending_queue:
+      enabled: true
+      num_consumers: 1
+      queue_size: 50
```

Comment:
```suggestion
    queue_size: 50
```

Rather than requiring the user to specify `enabled` and `num_consumers`, which only have a single valid value each (which is, sadly, not the zero value for their types), can you just have a queue size config option and add it to the `QueueSettings` struct initialization?

Reply:
Sounds good, I just realised that for myself also... https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/4698#discussion_r690651143

Reply:
But should it be `sending_queue.queue_size` or just `queue_size`? To me it makes sense to match other exporters, and error message in https://github.com/open-telemetry/opentelemetry-collector/blob/99e75af14ad3e5562962e5f5e647f7193de1ea03/exporter/exporterhelper/queued_retry.go#L190-L193 refers to  `sending_queue.queue_size` as well.